### PR TITLE
Display step progress when there is a timeout

### PR DESCRIPTION
**Pull request by alexandrudanpop**
Fixes OPS-2038

Mege first: https://github.com/openops-cloud/openops/pull/878
(as with this PR, we have live progress for runs, we can now display all step run status until the timeout)

## Testing Checklist

Check all that apply:

- [x] I tested the feature thoroughly, including edge cases

- [x] I verified all affected areas still work as expected

- [x] Automated tests were added/updated if necessary

- [x] Changes are backwards compatible with any existing data, otherwise a migration script is provided

## Visual Changes (if applicable)
image
https://www.loom.com/share/5e53443236c345e2949ed929725b145b

### CODE REVIEW

File: packages/react-ui/src/app/features/builder/run-details/run-details-helpers.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { FlowRun, FlowRunStatus, isNil } from '@openops/shared';
+import { t } from 'i18next';
+
+export function getRunMessage(
```
Comment (alexandrudanpop) [inline packages/react-ui/src/app/features/builder/run-details/run-details-helpers.ts:4]: extracted from `run-details/index.tsx` and  changed line 14 to not check TIMEOUT status. Previously:
```ts
  if (
    [FlowRunStatus.INTERNAL_ERROR, FlowRunStatus.TIMEOUT].includes(run.status)
  ) {
    return t('There are no logs captured for this run.');
  }
```

File: packages/react-ui/src/app/features/builder/run-details/run-details-helpers.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  if (isNil(run.logsFileId)) {
+    return t(
+      'Logs are kept for {days} days after execution and then deleted.',
+      { days: retentionDays },
+    );
+  }
+  return null;
+}
+
+export function getStatusText(status: FlowRunStatus, timeout: number): string {
```
Comment (alexandrudanpop) [inline packages/react-ui/src/app/features/builder/run-details/run-details-helpers.ts:26]: extracted as it was from `flow-runs/components/run-details-bar.tsx`

File: packages/server/shared/src/lib/system/system.ts [hunk 3]
```diff
@@ -62,7 +62,7 @@ const systemPropDefaultValues: Partial<Record<SystemProp, string>> = {
   [SharedSystemProp.PACKAGE_ARCHIVE_PATH]: 'cache/archives',
   [SharedSystemProp.BLOCKS_SOURCE]: BlocksSource.FILE,
   [AppSystemProp.QUEUE_MODE]: QueueMode.REDIS,
-  [SharedSystemProp.FLOW_TIMEOUT_SECONDS]: '600',
+  [SharedSystemProp.FLOW_TIMEOUT_SECONDS]: '60',
```
Comment (alexandrudanpop) [inline packages/server/shared/src/lib/system/system.ts]: remove as this is just for testing

File: packages/server/shared/src/lib/job/index.ts [hunk 4]
```diff
@@ -99,7 +99,7 @@ export const ResumeRunRequest = DelayedJobData;
 export type ResumeRunRequest = Static<typeof ResumeRunRequest>;
 
 export const flowTimeoutSandbox =
-  system.getNumber(SharedSystemProp.FLOW_TIMEOUT_SECONDS) ?? 600;
+  system.getNumber(SharedSystemProp.FLOW_TIMEOUT_SECONDS) ?? 60;
```
Comment (alexandrudanpop) [inline packages/server/shared/src/lib/job/index.ts]: remove as this is just for testing

**Review by cezudas — APPROVED** [review]
